### PR TITLE
After PR 776  - 830 redesign programs list

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/Pages/Collection/Page.js [hunk 1]
```diff
@@ -33,8 +32,7 @@ CollectionPage.propTypes = {
   collection: shapes.GenericCollection,
   wip: shapes.WIP,
   err: shapes.Error,
-  t: PropTypes.func.isRequired,
-  renderUnit: PropTypes.func.isRequired,
+  t: PropTypes.func.isRequired
```
Comment (gshilin) [inline src/components/Pages/Collection/Page.js:35]: I see renderUnit is still parameter. Why did you remove it here?

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Pages/Collection/Page.js (hunk 1 above)
Comment (davgur) [reply src/components/Pages/Collection/Page.js:35]: removed

**Review by gshilin — APPROVED** [review]
